### PR TITLE
docs: Windows instructions on running streamr-docker-dev

### PR DESCRIPTION
**Pull request by jtakalai**
We got it running with Swash guys! It's not using the script, maybe it could be tested with WSL, too, or smth.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,16 @@ ln -sf $(pwd)/streamr-docker-dev/bin.sh /usr/local/bin/streamr-docker-dev
 
 The docker stack has not been tested in a Windows environment and is not recommended at this time.
 
+If you know what services you need, you don't need to use the `bin.sh`, you can just use `docker-compose` directly, like so:
+
+1. Install and start Docker service.
+
+2. Clone this repo: `git clone git@github.com:streamr-dev/streamr-docker-dev.git`, change into that directory `cd streamr-docker-dev`
+
+3. Bind the loopback interface to 10.200.10.1: `netsh int ip add address "Loopback" 10.200.10.1`
```
Comment (kare) [inline README.md:49]: Should the `bin.sh` script be updated with this?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
   </a>
 </p>
 <h1 align="left">
-   
+
```
Comment (kare) [inline README.md:7]: I'd prefer to have a separate commit/PR for these formats. While it's not a major issue here, It is confusing.

File: README.md (hunk 1 above)
Comment (jtakalai) [reply README.md:49]: after there is some idea how to run .sh scripts in windows. Needs more research, maybe it's WSL, maybe something else. Maybe there should be a .bat version of that script. I don't know.

File: README.md (hunk 2 above)
Comment (jtakalai) [reply README.md:7]: my IDE does it automatically

Comment (kare) [reply README.md:7]: @jtakalai Then just commit the formats first

**Review by kare — APPROVED** [review]
LGTM

**Pull request merged**